FrontEnd/src/stores/program.js

```diff
@@ -0,0 +1,92 @@
+import { defineStore } from "pinia";
+// Import axios to make HTTP requests
+
+import { HTTP } from "../../http-common";
+
+export const useProgramStore = defineStore("program", {
+  state: () => ({
+    programs: [],
+    currentprogram: "",
+    supplement:[],
+    donor:[]
+  }),
+  getters: {
+    getProgram(state) {
+      return state.programs;
+    },
+  },
+  actions: {
+    async fetchProgram() {
+      try {
+        const data = await HTTP.get("api/get_programs");
```

Comment:
we can remove api/ here as it will be available in http-common file itself. so you can remove the api in all the below endpoint calls also.